### PR TITLE
Recreated shifts history page

### PR DESCRIPTION
**Pull request by MyIgel**
Re-implementation of the shifts history page, updated some used translations

### CODE REVIEW

File: resources/views/admin/shifts/history.twig [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                        <td>{{ shift.count }}</td>
+                        <td>{{ shift.start.format(__('Y-m-d H:i')) }}</td>
+                        <td>{{ shift.end.format(__('Y-m-d H:i')) }}</td>
+                        <td>{{ m.user(shift.createdBy) }}</td>
+                        <td>{{ shift.created_at.format(__('Y-m-d H:i')) }}</td>
+                        <td>
+                            <form method="post" class="ps-1">
+                                {{ csrf() }}
+                                {{ f.hidden('transaction_id', shift.transaction_id) }}
+                                {{ f.button(m.icon('trash'), {
```
Comment (xuwhite) [inline resources/views/admin/shifts/history.twig:53]: I think it's risky to not have a "are you sure you want to delete this" thing

File: includes/view/UserAngelTypes_view.php [hunk 2]
```diff
@@ -153,7 +153,7 @@ function UserAngelType_add_view(AngelType $angeltype, $users_source, $user_id)
         form([
             form_info(__('Angeltype'), $angeltype->name),
             form_checkbox('auto_confirm_user', __('Confirm user'), true),
-            form_select('user_id', __('User'), $users, $user_id),
```
Comment (xuwhite) [inline includes/view/UserAngelTypes_view.php:156]: Pls delete the old and now not used translations

File: includes/pages/admin_shifts.php [hunk 3]
```diff
@@ -564,99 +560,3 @@ function admin_shifts()
...
-    return page_with_title(admin_shifts_history_title(), [
-        msg(),
-        table([
-            'transaction_id' => __('ID'),
-            'title'          => __('title.title'),
-            'count'          => __('Count'),
-            'start'          => __('Start'),
-            'end'            => __('End'),
-            'user'           => __('User'),
-            'created_at'     => __('Created'),
```
Comment (xuwhite) [inline includes/pages/admin_shifts.php:658]: Pls delete the old and now not used translations

**Review by xuwhite — CHANGES_REQUESTED** [review]

File: includes/view/UserAngelTypes_view.php (hunk 2 above)
Comment (xuwhite) [reply includes/view/UserAngelTypes_view.php:156]: Done

File: resources/views/admin/shifts/history.twig (hunk 1 above)
Comment (MyIgel) [reply resources/views/admin/shifts/history.twig:53]: That requires #1247

**Comment by xuwhite** [comment]
looks fine

**Review by msquare — APPROVED** [review]

**Pull request merged**